### DISCUSSION TITLE
I don't have access to Codespaces

### DISCUSSION THREAD

**Discussion by TechStudent10** [community/community#3710] [Codespaces] [answered]
So around 2 weeks ago, I signed up for the Codespaces Beta, and I still don't have access to it! Why don't I? I haven't tried again 'cause it said, "Don't do anything, just monitor your primary email" or something along those lines.

**Comment by 2percentsilk** [comment]
Thanks for your feedback. We know there's a lot of excitement for the beta (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We'll add more folks as soon as we can!

**Reply by ghost** [reply]
Yay!

**Reply by TechStudent10** [reply]
> We know there's a lot of excitement for the beta (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight.

Well okay. Thanks!

**Reply by TechStudent10** [reply]
@2percentsilk alright it's been days and I still don't have access to Codespaces.

**Reply by 2percentsilk** [reply]
@TechStudent11 I'm sorry but we can't provide any exact ETAs for access.
We're excited to see so much interest from the community but unfortunately, that means there's quite a backlog of folks to add and we can't provide an estimate for you specifically can expect access. If you're account is already on the waiting list, we simply ask for your patience at this time.

**Reply by TechStudent10** [reply]
@2percentsilk Alright...

**Reply by ladeak** [reply]
@2percentsilk is (previous) Visual Studio Codespaces suggeseted to be used in the meanwhile? Is there going to be a migration path from VS Codespaces to github codespaces?

**Reply by TechStudent10** [reply]
@2percentsilk it's now been 3 weeks and Codespaces still isn't working. Should I signup again?

**Reply by 2percentsilk** [reply]
@ladeak Visual Studio Codespaces was retired in February 2021. If you were a current VS Codespaces user, you should have received instructions in product and via email on migrating.

@TechStudent11 no need to sign-up again. When you're granted access, you'll get an email letting you know. However, long wait times are normal as we are being cautious with the rollout and it's currently a limited beta.

**Reply by TechStudent10** [reply]
@2percentsilk okay. Thanks!

**Comment by ajsirota** [comment]
I feel the same way. I signed up for the waitlist a few months ago, and I still don't have access to the beta. If we could have more transparency on when we will gain access, that would be much appreciated.

**Reply by 2percentsilk** [reply]
Hi @ajs256,

I understand your frustration but at this time, I can't provide an ETA for when to expect access. There's a lot of interest in the beta (which we're very thankful for!) but we're trying to be cautious in our rollout. As such, we ask for your patience at this time.

**Comment by TechStudent10** [comment]
Alright, Codespaces has been released to all now, but it's paid. Aaaaaaaaaand it's only for Teams. Aaaaaaaaaaaand it's still in beta for individuals. Oh well...

**Reply by ladeak** [reply]
I am afraid it is only available for teams yet:

https://github.com/features/codespaces

> Codespaces is rolling out progressively on August 11th, 2021 and can be enabled in settings by organization owners for Team and Enterprise Cloud plans. For users in individual plans, we’re extending the existing Codespaces beta. For those in the beta, access will remain and we’ll share updates on what’s coming in the near future.

**Reply by TechStudent10** [reply]
@ladeak ohhh ok

**Reply by ladeak** [reply]
Also mentioned on the top of the quickstart: https://docs.github.com/en/codespaces/getting-started/quickstart

**Reply by TechStudent10** [reply]
@ladeak okay.

**Comment by TechStudent10** [comment, marked as answer]
Over 1 YEAR after applying for the Codespaces Beta, I finally got it...

![image](https://user-images.githubusercontent.com/76978184/169706191-b0ee60ca-6697-4117-809f-29ddc6477a1b.png)

**Reply by Milo123459** [reply]
I got it on the same day haha

**Reply by TechStudent10** [reply]
Awesome!

**Reply by chunkybanana** [reply]
Same :), it's cool but annoying

**Reply by KendallDoesCoding** [reply]
You know what I did, I applied and then reapplied and then got instant access it's a glitch lol

**Reply by KendallDoesCoding** [reply]
> Did it get any better?

I believe so

**Reply by KendallDoesCoding** [reply]
There was a glitch/bug at that time lol

**Reply by JacobwLyman** [reply]
Where can I register for the waitlist list? I can't find it anywhere!

**Reply by TechStudent10** [reply]
> Where can I register for the waitlist list? I can't find it anywhere!

@JacobwLyman Codespaces is out of beta. I think this is just GitHub clearing out beta-testers so sadly you can't apply for the beta for free and have to pay for normal Codespaces.

**Comment by KendallDoesCoding** [comment]
Google. Codespaces waitlist... i think the url is github.com/codespaces but
idk

**Reply by hidp123** [reply]
I can't find the link, did you find it?

**Comment by KendallDoesCoding** [comment]
So the people who signed up for beta will not have to pay? Or they may do
something like Copilot.. ppl with github student plan will not have to pay?
let's see.
